syncopy/nwanalysis/connectivity_analysis.py

```diff
@@ -4,32 +4,30 @@
 #
 
 # Builtin/3rd party package imports
-import numpy as np
-
-# Syncopy imports
-from syncopy.shared.parsers import data_parser, scalar_parser
-from syncopy.shared.tools import get_defaults, best_match, get_frontend_cfg
-from syncopy.datatype import CrossSpectralData, AnalogData, SpectralData
-from syncopy.shared.errors import (
-    SPYValueError,
-    SPYWarning,
-    SPYInfo)
-from syncopy.shared.kwarg_decorators import (unwrap_cfg, unwrap_select,
-                                             detect_parallel_client)
+from syncopy.nwanalysis.AV_compRoutines import NormalizeCrossSpectra, NormalizeCrossCov, GrangerCausality
+from syncopy.nwanalysis.ST_compRoutines import CrossSpectra, CrossCovariance, SpectralDyadicProduct
 from syncopy.shared.input_processors import (
     process_taper,
     process_foi,
     process_padding,
     check_effective_parameters,
     check_passed_kwargs
 )
-
-from syncopy.nwanalysis.ST_compRoutines import CrossSpectra, CrossCovariance, SpectralDyadicProduct
-from syncopy.nwanalysis.AV_compRoutines import NormalizeCrossSpectra, NormalizeCrossCov, GrangerCausality
+from syncopy.shared.kwarg_decorators import (unwrap_cfg, unwrap_select,
+                                             detect_parallel_client)
+from syncopy.shared.errors import (
+    SPYValueError,
+    SPYWarning,
+    SPYInfo)
+from syncopy.datatype import CrossSpectralData, AnalogData, SpectralData
+from syncopy.shared.tools import get_defaults, best_match, get_frontend_cfg
+from syncopy.shared.parsers import data_parser, scalar_parser
+import numpy as np
+# Syncopy imports
 
 
-availableMethods = ("coh", "corr", "granger")
-coh_outputs = {"abs", "pow", "complex", "fourier", "angle", "real", "imag"}
+availableMethods = ("coh", "corr", "granger", "csd")
+connectivity_outputs = {"abs", "pow", "complex", "fourier", "angle", "real", "imag"}
```

Comment:
I am ok with the renaming, but then you also have to change the name in the respective `TestCoherence`, this is why the pipeline fails atm :wink: